backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/AnsibleCallback.java

```diff
@@ -59,8 +57,6 @@ public void doPolling(Guid cmdId, List<Guid> childCmdIds) {
             }
         };
 
-        AnsibleReturnValue ret = new AnsibleReturnValue(AnsibleReturnCode.ERROR);
-        ret.setLogFile(runnerClient.getLogger().getLogFile());
```

Comment:
are you sure this is not used in processEvents() by using that singleton's property ?
I don't think we can skip that, AFAIU this is the only reason e.g. https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java#L126 is logging into the right file (most of the time:)

Reply:
ah, sorry, that's set by line 46

Reply:
> are you sure this is not used in processEvents() by using that singleton's property ?

I only modified `processEvents` method's signature - there were no signs of usage at all.

> I don't think we can skip that, AFAIU this is the only reason e.g. https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java#L126 is logging into the right file (most of the time:)

Isn't [this line](https://github.com/oVirt/ovirt-engine/blob/3e0b763e86b66bd4cfaf4b2ebaf8f04a5e003d5a/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/AnsibleCallback.java#L48) setting the appropriate file?

Reply:
yes, i mixed up links:)